File: md-finances/src/webStatic/apps/old/oldApps/Common/data/TaxationSystemType.js
window.Common = (function (common) {

    common.Data = common.Data || {};

    common.Data.TaxationSystemType = {
        Default: 0,
        Usn: 1,
        Osno: 2,
        Envd: 3,
        UsnAndEnvd: 4,
        OsnoAndEnvd: 5,
        Patent: 6
    };
    
    return common;
    
})(window.Common);

File: md-money/md-accountingStatements-shared/src/Moedelo.AccountingStatements.Enums/TaxationSystemType.cs
namespace Moedelo.AccountingStatements.Enums
{
    /// <summary>
    /// Система налогообложения
    /// </summary>
    public enum TaxationSystemType
    {
        /// <summary>
        /// УСН
        /// </summary>
        Usn = 1,

        /// <summary>
        /// ОСНО
        /// </summary>
        Osno = 2,

        /// <summary>
        /// ЕНВД
        /// </summary>
        Envd = 3,

        /// <summary>
        /// УСН + ЕНВД
        /// </summary>
        UsnAndEnvd = 4,

        /// <summary>
        /// ОСНО + ЕНВД
        /// </summary>
        OsnoAndEnvd = 5,

        /// <summary>
        /// Патент
        /// </summary>
        Patent = 6
    }
}
